examples/weather-demo/Cargo.toml

```
@@ -0,0 +1,73 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+[package]
+name = "rusty-weather"
+version = "1.0.0"
```

I think we usually match the version with the rest of Slint in the repo :)

```suggestion
version = "1.7.0"
```

---

Done